### DISCUSSION TITLE
blog/posts/open-folder-in-vscode-with-a-hotkey-or-context-menu-in-macos

### DISCUSSION THREAD

**Discussion by giscus** [pavlovtech/NextBlog#10] [General]
# blog/posts/open-folder-in-vscode-with-a-hotkey-or-context-menu-in-macos

I’m used to the Open in VSCode context menu action in Windows, but it was missing in MacOS. So I found how to add it and want to share it.

https://www.alexpavlov.dev/blog/posts/open-folder-in-vscode-with-a-hotkey-or-context-menu-in-macos

**Comment by alphaoumardev** [comment]
grtger
